dockstore-webservice/src/main/java/io/dockstore/webservice/core/ParsedInformation.java

```diff
@@ -0,0 +1,51 @@
+package io.dockstore.webservice.core;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import io.dockstore.common.DescriptorLanguage;
+
+/**
+ * This is for information gained after parsing the workflow with a language parser (WDLHandler, CWLHandler, etc)
+ *
+ * Putting DescriptorLanguage here instead of using a map with DescriptorLanguage because there may be cases where the language is
+ * irrelevant. For example, workflows only have one language.
+ */
+@Entity
+public class ParsedInformation {
+    @Id
+    @GeneratedValue
+    private Long id;
```

Comment:
Why is this Long instead of long? I think Long should only be used if it's nullable.